ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -162,10 +164,22 @@ export default defineComponent({
       localState.dataLoaded = true
     })
 
+    // Future state to parse all relevant MHR Information
     const parseMhrInformation = async (): Promise<void> => {
-      // Future state to parse all relevant MHR Information
+      await parseCurrentOwnerGroups()
+    }
+
+    const parseCurrentOwnerGroups = async (): Promise<void> => {
       const { data } = await fetchMhRegistration(getMhrInformation.value.mhrNumber)
       const currentOwnerGroups = data?.ownerGroups || [] // Safety check. Should always have ownerGroups
+      // Create an ID to each individual owner for UI Tracking
+      currentOwnerGroups.forEach(ownerGroup => {
```

Comment:
So this is purely for UI? `MhrRegistrationHomeOwnerIF` has id but it's optional, maybe we should make it required, so we don't have to set it here?

Reply:
Required as in filed with an ID, preserved in the DB and returned?
Via MHRegistrations?

Reply:
But yes, it does seem that it SHOULD be required, for UI at least. Since we do require it for functionality.

Reply:
Resolving after discussion.